llvm/tools/llvm-exegesis/lib/BenchmarkResult.cpp

```diff
@@ -258,6 +258,64 @@ template <> struct ScalarTraits<exegesis::RegisterValue> {
   static const bool flow = true;
 };
 
+template <> struct ScalarTraits<APInt> {
```

Comment:
I think you're only supposed to specialize this for types you own. @gchatelet wrote the yaml input/output, can you have a look ?